zuul.d/test_jobs.yaml

```diff
@@ -0,0 +1,15 @@
+- job:
```

Comment:
Please drop this file and also fix the commit message.

Reply:
done